python/api-examples-source/status.toast3.py

```diff
@@ -0,0 +1,10 @@
+import streamlit as st
+import time
+
+go = st.radio('Give a toast?', ['Yes','No'])
+
+if go == 'No': st.stop()
+
+with st.toast('Cheers!'):
+    time.sleep(2)
+    st.balloons()
```

Comment:
What does the `with` notation accomplish? Haven't gone through the spec in detail.

Selecting 'Yes' displays the toast, but not the balloons 🤔 This example could be revised. The other two are good to merge!

Reply:
Yes, this is a placeholder app for the next release, but I can remove it. (See project channel in Slack.)